examples/contrib/oed/eig_estimation_benchmarking.py

```diff
@@ -0,0 +1,82 @@
+import time
```

Comment:
is this supposed to work? when i run this i get "UserWarning: Encountered NaN: loss"

Reply:
Yeah I saw this once before. What happens if you run it again?

Reply:
ah, seems to be a pytorch 0.4.1 versus 0.4 problem.

@neerajprad @fritzo if you're looking for 0.4.1/0.4 discrepancies try

pytest -s -v examples/contrib/oed/eig_estimation_benchmarking.py 

works fine in 0.4 but i get "UserWarning: Encountered NaN: loss" in 0.4.1

no idea why

Reply:
Hmmm... I'll stick to 0.4 for local testing for now